tests/xdsl_opt/test_xdsl_opt.py

```diff
@@ -1,18 +1,18 @@
-import pytest
-from xdsl.xdsl_opt_main import xDSLOptMain
 from contextlib import redirect_stdout
 from io import StringIO
-import os
-from xdsl.ir import MLContext
+
+import pytest
+
 from xdsl.dialects.builtin import ModuleOp
-from xdsl.rewriter import Rewriter
+from xdsl.ir import MLContext
+from xdsl.xdsl_opt_main import xDSLOptMain
 
 
 def test_opt():
     opt = xDSLOptMain(args=[])
     assert list(opt.available_frontends.keys()) == ['xdsl', 'mlir']
     assert list(opt.available_targets.keys()) == ['xdsl', 'irdl', 'mlir']
-    assert list(opt.available_passes.keys()) == []
+    assert list(opt.available_passes.keys()) == ['lower-mpi']
```

Comment:
At some point we might want to think about a naming scheme for rewrites, but that is something for the distant future, IMO.